modules/op-release-notes-1-8.adoc

```diff
@@ -533,3 +533,69 @@ securityContext:
+
+
+[id="release-notes-1-8-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.1
+
+With this update, {pipelines-title} General Availability (GA) 1.7.2 is available on {product-title} 4.9, 4.10, and the upcoming version.
+
+[id="known-issues-1-8-1_{context}"]
+=== Known issues
+
+* When installer sets associated with cluster tasks are in failed state, the status of the `TektonConfig` custom resource is incorrectly displayed as `True` instead of `False`.
++
+.Example: Failed installer sets
+[source,terminal]
+----
+$ oc get tektoninstallerset 
+NAME                                     READY   REASON
+addon-clustertasks-nx5xz                 False   Error
+addon-communityclustertasks-cfb2p        True    
+addon-consolecli-ftrb8                   True    
+addon-openshift-67dj2                    True    
+addon-pac-cf7pz                          True    
+addon-pipelines-fvllm                    True    
+addon-triggers-b2wtt                     True    
+addon-versioned-clustertasks-1-8-hqhnw   False   Error
+pipeline-w75ww                           True    
+postpipeline-lrs22                       True    
+prepipeline-ldlhw                        True    
+rhosp-rbac-4dmgb                         True    
+trigger-hfg64                            True    
+validating-mutating-webhoook-28rf7       True 
+----
++
+.Example: Incorrect `TektonConfig` status
+[source,terminal]
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.8.1     True 
+----
+
+// https://issues.redhat.com/browse/SRVKP-2556
+
+[id="fixed-issues-1-8-1_{context}"]
+=== Fixed issues
+
+* Before this update, the pruner deleted task runs of running pipelines and displayed the following warning: `some tasks were indicated completed without ancestors being done`. With this update, the pruner retains the task runs that are part of running pipelines.
+// https://issues.redhat.com/browse/SRVKP-2419
+
+* Before this update, `pipeline-1.8` was the default channel for installing the {pipelines-title} Operator 1.8.x. With this update, `latest` is the default channel.
+// https://issues.redhat.com/browse/SRVKP-2412
+
+* Before this update, the {pac} controller pods did not have access to certificates exposed by the user. With this update, {pac} can now access routes and Git repositories behind a self-signed or a custom certificate.
+// https://issues.redhat.com/browse/SRVKP-2470
+
+* Before this update, the task failed with RBAC errors after upgrading from {[pipelines-title]} 1.7.2 to 1.8.0. With this update, the tasks run successfully without any RBAC errors.
+// https://issues.redhat.com/browse/SRVKP-2472
+
+* Before this update, using the `tkn` CLI tool cluster administrators could not remove the task runs and pipline runs that contained a `result` object of `array` type. With this update, cluster adminstrators can use the `tkn` CLI tool to remove the task runs and pipeline runs that contain a `result` object of `array` type.
```

Comment:
This is the case for the normal user also and not only with the administrator right?
@concaf @rupalibehera

Reply:
Yes @VeereshAradhya